YumemiTraining/YumemiTraining/ViewController.swift

```diff
@@ -24,12 +26,44 @@ final class ViewController: UIViewController {
     private func loadWeather() {
         
         do {
+            // 天気予報に必要なパラメータをJSON文字列で保持
+            let jsonObj: [String: Any] = [
+                "area": "tokyo",
+                "date": "2020-04-01T12:00:00+09:00"
+            ]
+            
+            let jsonData = try JSONSerialization.data(withJSONObject: jsonObj, options: [])
+            guard let jsonStr = String(data: jsonData, encoding: .utf8) else {
+                throw LoadWeatherError.decodeDataFailed
+            }
+            
+            // 天気予報をAPIから取得
+            let weatherJsonStr = try YumemiWeather.fetchWeather(jsonStr)
+            guard let weatherData: Data =  weatherJsonStr.data(using: String.Encoding.utf8) else {
```

Comment:
@torakyun 
すいません、前回指摘し忘れましたが不要な型の記載は行わなくて良いです。あとスペースが一つ多いようですね。
```suggestion
            guard let weatherData = weatherJsonStr.data(using: String.Encoding.utf8) else {
```

Reply:
https://github.com/torakyun/yumemi_training/pull/9/commits/5e48543c8ed8497a2929d173df73401cc97ebbd5
このコミットで修正しました。